### ISSUE TITLE
Rdkafka 5.0.0 supports librdkafka-dev package?

### ISSUE THREAD

**Issue by AratKruglik** [php-rdkafka/php-rdkafka#435]
Tying to install 5.0.0 version with my PHP8 docker config and I getting nex error:
```
WARNING: channel "pecl.php.net" has updated its protocols, use "pecl channel-update pecl.php.net" to update
downloading rdkafka-5.0.0.tgz ...
Starting to download rdkafka-5.0.0.tgz (41,427 bytes)
............done: 41,427 bytes
31 source files, building
running: phpize
Configuring for:
PHP Api Version:         20190902
Zend Module Api No:      20190902
Zend Extension Api No:   320190902
building in /tmp/pear/temp/pear-build-defaultuserI0Yu40/rdkafka-5.0.0
running: /tmp/pear/temp/rdkafka/configure --with-php-config=/usr/bin/php-config
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for PHP prefix... /usr
checking for PHP includes... -I/usr/include/php/20190902 -I/usr/include/php/20190902/main -I/usr/include/php/20190902/TSRM -I/usr/include/php/20190902/Zend -I/usr/include/php/20190902/ext -I/usr/include/php/20190902/ext/date/lib
checking for PHP extension directory... /usr/lib/php/20190902
checking for PHP installed headers prefix... /usr/include/php/20190902
checking if debug is enabled... no
checking if zts is enabled... no
checking for gawk... no
checking for nawk... nawk
checking if nawk is broken... no
checking for rdkafka support... yes, shared
checking for librdkafka/rdkafka.h" in default path... found in /usr
checking for rd_kafka_new in -lrdkafka... yes
checking for librdkafka version... configure: error: librdkafka version 0.11.0 or greater required.
ERROR: `/tmp/pear/temp/rdkafka/configure --with-php-config=/usr/bin/php-config' failed
```
How to solve it? Thanks!

**Comment by nick-zh** [comment]
`configure: error: librdkafka version 0.11.0 or greater required` seems your version is too old, so far i had no trouble building the master of librdkafka, the dev version gets recognized properly

**Comment by AratKruglik** [comment]
`librdkafka-dev` is installed

**Comment by nick-zh** [comment]
yeah, but it seems you are using a version below `0.11.x`

**Comment by arnaud-lb** [comment]
Hi @AratKruglik,

What is your `librdkafka-dev` version ?

This confirms that the library is indeed installed:

```
checking for librdkafka/rdkafka.h" in default path... found in /usr
checking for rd_kafka_new in -lrdkafka... yes
```

However, this suggests that the version installed in /usr is too old:

```
checking for librdkafka version... configure: error: librdkafka version 0.11.0 or greater required.
```

**Comment by andrewmy** [comment]
If you're installing on Debian or Ubuntu, make sure you're using the Confluent package repository as described on https://docs.confluent.io/platform/current/installation/installing_cp/deb-ubuntu.html

tl;dr:
```bash
echo "deb [arch=amd64] https://packages.confluent.io/deb/6.0 stable main" >> /etc/apt/sources.list
wget -qO - https://packages.confluent.io/deb/6.0/archive.key | apt-key add -
apt-get update
apt-get install -y librdkafka1 librdkafka-dev
```

**Comment by AratKruglik** [comment]
Yes, now it works! Thanks a lot for supporting!
